runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/net/IpV4Addr.kt

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package aws.smithy.kotlin.runtime.net
+
+/**
+ * An IPv4 address as defined by [RFC 791](https://www.rfc-editor.org/rfc/rfc791)
+ */
+public data class IpV4Addr(
+    /**
+     * The four eight-bit integers that make up this address
+     */
+    override val octets: ByteArray,
+) : IpAddr() {
```

Comment:
**Nit**: Missed this the first time but this is not the correct way to document parameters. They should be annotated alongside the class/method documentation:

```kotlin
/**
 * An IPv4 address as defined by [RFC 791](https://www.rfc-editor.org/rfc/rfc791)
 * @param octets The four eight-bit integers that make up this address
 */
```

Doing it this way compiles but has no effect on IDE/HTML docs.

Same problem noted in `IpV6Addr`.